data/json/items/melee/bludgeons.json

```
@@ -1990,5 +1990,51 @@
     "price": "160 USD",
     "qualities": [ [ "HAMMER", 1 ] ],
     "melee_damage": { "bash": 22, "stab": 23 }
+  },
+  {
+    "type": "TOOL",
+    "id": "exodii_prybar",
+    "name": { "str": "breacher" },
+    "description": "Some sort of large prybar of a make and manufacturer you've never seen before.  On the heavier end is a set of thick, crabclaw like shears connected to a hydraulic piston, capable of cutting or prying through metal with great force.  The Exodii use these for general purpose breaching and scrapping heavy machines, and it's been retrofitted to run on their bionic power.",
+    "price_postapoc": "45 USD",
+    "color": "dark_gray",
+    "symbol": "/",
+    "material": [ "hc_steel", "ceramic" ],
+    "flags": [ "DURABLE_MELEE", "SHEATH_AXE", "USES_BIONIC_POWER" ],
```

I'm assuming this came from the halligan bar, not applicable.
```suggestion
    "flags": [ "DURABLE_MELEE", "USES_BIONIC_POWER" ],
```